docs/UserGuide.md

```diff
@@ -212,88 +212,102 @@ Adds an income of $200 from a gift, using today’s date and the default categor
 ### Edit Transaction
 Edits an existing transaction in your financial list.
 
-**Format**: `edit INDEX [/des DESCRIPTION] [/a AMOUNT] [/d DATE] [/c CATEGORY]`
+**Format**: `edit [INDEX] [/des DESCRIPTION] [/a AMOUNT] [/d DATE] [/c CATEGORY]`
 
- - Edits the transaction at the specified `INDEX`. `INDEX` refers to the index number shown in the displayed financial list when [`list`](#list-transactions) is called. 
- - `INDEX` must be a positive integer.
+ - Edits the the specific field(s) of a transaction.
+ - Should at least modify one field.
 
 **Parameter Details:** (Refer to [here](#param_details) for what each parameter represents)
-
+ - `INDEX` (optional):
+   - `INDEX` refers to the index number shown in the displayed financial list when [`list`](#list-transactions) is called (ordered by date).
+   - `INDEX` must be a positive integer.
+   - If the `INDEX` is omitted, then will edit the amended trasaction as default.
  - `DESCRIPTION` (optional): 
    - shouldn't be blank if provided.
-   - DO NOT USE `¦¦` in `DESCRIPTION` cause it serve as the seperator token in storage file.
+   - DO NOT USE `¦¦` and `/` in `DESCRIPTION` cause these symbols are used as separator tokens in the storage file and as prefixes for command arguments.
+   - We don't support characters outside of English alphabets and numbers.
  - `AMOUNT` (optional):
    - Must be a positive number with a maximum value of $9999999.00. If it's a floating-point number, it will be rounded to two decimal places.
  - `DATE` (optional):
-   - Should follow `DD/MM/YY` format and cannot be after the system date.
+   - Should follow `DD/MM/YYYY` format and cannot be after the system date.
  - `CATEGORY` (optional):
    - Should be one of the categories allowed in Expenses/Incomes.
 
 **Example Usages**:
-
+Example 1: Edits the description of the 1st entry to be `breakfast`.
```

Comment:
Perhaps you could add a newline here to prevent the header and the text line from being on the same line?